scenes/game.js

```diff
@@ -3,13 +3,14 @@ var TILE_WIDTH = 256;
 var isos = Crafty.diamondIso.init(TILE_WIDTH,TILE_HEIGHT,20,20);
 
 Crafty.defineScene("Game", function() {
-	Crafty.viewport.scale(0.3);
 
 	var ground = '2D, DOM, Color, tile';
 	var sand = '2D, DOM, Color, tile';
 	var urn = '2D, DOM, Color, urn';
 	var player = Crafty.e('Player');
 
+    makeCameraTrackEntity(player, 75);
```

Comment:
Camera tracker updates scale to 1.0, so we have to set scale after it; and camera trakcer has to be after the player is defined so hence scale 0.3 is now later.